ai-missions/high-risk-flu-shot-mission/skills/send_nurseline_notification/Readme.md

```diff
@@ -0,0 +1,34 @@
+### Send Nurseline Notification
+
+
+#### Description:
+For the given profile IDs will send notifications on preferred communication channel to call nurse line to know more information about flushot and flushot providers.
```

Comment:
Missing some words I think.  Change to:  For the Profile Ids derived from the configured Cohort, notifications are sent on the preferred communication channel that request that members call the nurse-line for more information about scheduling a flu shot with an available flu shot provider.